auth-jwt/auth-jwt-java-sbt/README.md

```diff
@@ -0,0 +1,48 @@
+# Lagom Recipe: How authenticate/authorize by JWT
+
+This recipe demonstrates, how you can use [JWT](https://en.wikipedia.org/wiki/JSON_Web_Token) 
+for protection your service endpoints. It used library [PAC4J](https://www.pac4j.org/) 
+and her [module](https://github.com/pac4j/lagom-pac4j) of integration with Lagom. 
+
+## Testing the recipe
+
+#### unit tests
+
+You can test this recipe using the provided tests:
+
+```bash
+sbt test
+```
+
+#### manual tests
+
+You can also test this recipe manually using 2 separate terminals.
+
+On one terminal start the service:
+
+```bash
+sbt runAll
+```
+
+On a separate terminal, use `curl` to send:
+
+* anonymous request
+```bash
+curl http://localhost:9000/authenticate
+```
+
+* request from Alice
+```bash
+curl -H "Authorization: Bearer eyJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJBbGljZSIsInJvbGVzIjpbIm1hbmFnZXIiXSwiaXNzIjoiaHR0cHM6XC9cL3BhYzRqLm9yZyIsImlhdCI6MTU0NDI4OTkxNywianRpIjoiODA4M2U0MGUtMmU4YS00YzVhLThlMTEtNDBmZjEyNzE3MjhkIn0.XlwqU6JHKeGf3MNVFfhMXT5MZKxcjobAffnTRUam1WVNqwOu0MLdmYiu0HIqBsytPDEQfRUj_Qkp62v0yEarKMuLTX3rVwZkh07B4uDwHrB5Q54la70F8N1wLfo1RzDKvCljtlSgAG7IG0VHjEqsTSEF57J1S5mhDmlXCqAtDaO3OhMbQexXxxowOC1UKNFjIPRoSwzgIFIv6fyXuMbm7WPiuEKWPI9RAod_VAoety0QZV6Agk5gTDi6ZxAU8iQqx1HHourwzaWBZdBS2rG8Bw8ltnfKsYe8Dq39K_Ca5srd3wTYBycUesGC7AhJoqI5AtrUuv0yoIQNWYZ1r8aCTg" http://localhost:9000/authenticate
+```
+
+* request from Alice to protected path 
+```bash
+curl -H "Authorization: Bearer eyJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJBbGljZSIsInJvbGVzIjpbIm1hbmFnZXIiXSwiaXNzIjoiaHR0cHM6XC9cL3BhYzRqLm9yZyIsImlhdCI6MTU0NDI4OTkxNywianRpIjoiODA4M2U0MGUtMmU4YS00YzVhLThlMTEtNDBmZjEyNzE3MjhkIn0.XlwqU6JHKeGf3MNVFfhMXT5MZKxcjobAffnTRUam1WVNqwOu0MLdmYiu0HIqBsytPDEQfRUj_Qkp62v0yEarKMuLTX3rVwZkh07B4uDwHrB5Q54la70F8N1wLfo1RzDKvCljtlSgAG7IG0VHjEqsTSEF57J1S5mhDmlXCqAtDaO3OhMbQexXxxowOC1UKNFjIPRoSwzgIFIv6fyXuMbm7WPiuEKWPI9RAod_VAoety0QZV6Agk5gTDi6ZxAU8iQqx1HHourwzaWBZdBS2rG8Bw8ltnfKsYe8Dq39K_Ca5srd3wTYBycUesGC7AhJoqI5AtrUuv0yoIQNWYZ1r8aCTg" http://localhost:9000/authorize
+```
+
+* request from Bob to protected path 
+```bash
+curl -H "Authorization: Bearer eyJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJCb2IiLCJyb2xlcyI6WyJkZXZlbG9wZXIiXSwiaXNzIjoiaHR0cHM6XC9cL3BhYzRqLm9yZyIsImlhdCI6MTU0NDI4OTkxNywianRpIjoiMDEzNzgwOTgtYmE1ZS00OTRlLTlhMzktYTA4MTE1NjNmZWVjIn0.fxRG69CgM43Xpg2t5I-9qmGdCzNOrtWb9F1wS4BbcxuRGIp9MC5wYTO-_aASc9DozJooVG96P7aQqy6qPeg31NjVIfYghLzWOM9ZO0cewkUhzC2FKSt0ZdXAfLhwPl5dtx0Q0mhAK6z-dnCpmgkSGazg_lt6OFuDNr-3laNt7y_0mVm5-zgrRVjGTMKLLbkmyofFr6eSjxl-Cc4wOOqDYDfJvQzTe4bjenBx-32GsZORidxGuGSU7RJftWdN1tZFSemlZo-blZIsW2vcIgEEY9sdQDg06MDREUaDwTiA_0jfQqUjX0jqsAetuGPVKuOM_0QrPj-CEh6QIbu-AtSN4A" http://localhost:9000/authorize
+```
+
```

Comment:
I think it will be useful to mention in that section what is the expected output for each of this calls. 

For instance, Bob is not authorized. At least, that's what I got and I don't know if that's the expected behaviour.

Reply:
Thanks 👍 
I added to README:
* a bit info about methods
* roles of Alice and Bob
* output for all `curl` calls